opentracing-api-extensions/src/main/java/io/opentracing/contrib/api/SpanData.java

```diff
@@ -33,6 +33,10 @@
      */
     Object getCorrelationId();
 
+    String getTraceId();
```

Comment:
Shouldn't this be `toTraceId()`, for consistency?

Reply:
Here is the official statement why it's called `toTraceId`:

> The method names `ToTraceID` and `ToSpanID` were chosen over `TraceID` and `SpanID` for two reasons:
>
> 1) To avoid conflicts with tracer implementations, which often use `TraceID` and `SpanID` to expose their identifiers in their native format.
> 2) To communicate to the user that a format conversion may be occuring, and thus there may be some cost to calling these methods.
>
> https://github.com/opentracing/specification/blob/master/rfc/trace_identifiers.md#why-totraceid-instead-of-traceid

The conversion already happened so I'd argue that it's actually more consistent with the spec to use `getTraceId` here.